### PR TITLE
Add openSUSE installation steps

### PR DESCRIPTION
**Pull request by hwsmm**
Not so sure about formatting, but I guess that's it?

Related to #40. There are a few more RPM-based distros, but I am not sure if we need to cover more of them.

### CODE REVIEW

File: site/_wiki/Install/Linux.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ If you have the Microsoft dotnet repository installed you will have to either ma
 
 If you're experiencing `libhostfxr` issues, please see the solutions from Microsoft [here]({{ site.data.links.external.Microsoft.DotnetLinuxPackageMixup }}).
 
-## Fedora {#rpm}
```
Comment (gonX) [inline site/_wiki/Install/Linux.md:27]: Should probably make a new section here instead with a title like "RPM-based distributions" and then use H3's for Fedora and OpenSuSE respectively instead

**Review by gonX — CHANGES_REQUESTED** [review]
Thanks, formatting seems fine, otherwise the tests should tell us. Just some styling concerns remaining

File: site/_wiki/Install/Linux.md [hunk 2]
```diff
@@ -37,6 +39,27 @@ If you're experiencing `libhostfxr` issues, please see the solutions from Micros
...
+    > You don't need to install .NET runtime at this point as zypper will install it as a dependency of OpenTabletDriver.
+
+3. Run the following commands in a terminal
+
+    ```bash
+    # Update the package list
+    sudo zypper refresh
+
+    # Install the package
+    sudo zypper --no-gpg-check install ./OpenTabletDriver.rpm
```
Comment (jamesbt365) [inline site/_wiki/Install/Linux.md:56]:
```suggestion
    sudo zypper --no-gpg-checks install ./OpenTabletDriver.rpm
```

**Review by jamesbt365 — CHANGES_REQUESTED** [review]

**Review by gonX — APPROVED** [review]
LGTM

**Pull request merged**